### ISSUE TITLE
MagickNET logs too much events

### ISSUE THREAD

**Issue by zygimantaskazlauskas** [dlemstra/Magick.NET#24]
Hello,
I am trying to configure MagickNET log like this:
```
MagickNET.SetLogEvents(LogEvents.Cache | LogEvents.Exception | LogEvents.Resource);
MagickNET.Log += MagickNET_Log;
```

But in the actual log file i see all my specified events from enum + **Pixel, Coder, Policy, Blob** events.

How can i prevent these events from logging?

**Comment by dlemstra** [comment]
I can reproduce your issue and this will be resolved in the next version of Magick.NET that I will publish in a couple of days.

**Comment by dlemstra** [comment]
@zygimantaskazlauskas Can you confirm this is fixed in the new version of Magick.NET?

**Comment by zygimantaskazlauskas** [comment]
I can confirm that this issue is fixed in latest version (7.0.3.901).
Thank you @dlemstra.
